### PR TITLE
Update to use go 1.16

### PR DESCRIPTION
**Pull request by wallyworld**
Change the version of Go to 1.16

This is simply the first step - the Jenkins infrastructure also needs to be updated since the merge check tests etc still use 1.14

## QA steps

Unit tests

### CODE REVIEW

**Review by hpidcock — APPROVED** [review]

**Comment by wallyworld** [comment]
$$merge$$

**Comment by hpidcock** [comment]
$$merge$$

**Comment by SimonRichardson** [comment]
$$merge$$

File: worker/httpserver/worker.go [hunk 1]
```diff
@@ -175,13 +175,14 @@ func (w *Worker) loop() error {
 		}
 	}()
 	defer func() {
+		// Release the holdable listener to unblock any pending accepts.
+		// This needs to be done before asking to server to shutdown since
```
Comment (komuw) [inline worker/httpserver/worker.go:179]:
```suggestion
		// This needs to be done before asking the server to shutdown since
```

**Pull request merged**